src/v/config/node_config.cc

```diff
@@ -161,6 +161,13 @@ node_config::node_config() noexcept
       "The file name for the debug bundle zip file (default: debug-bundle.zip)",
       {.visibility = visibility::user},
       "debug-bundle.zip")
+  , rpk_path(
+      *this,
+      "rpk_path",
+      "The path to the RPK binary. Used to create debug bundles (default: "
+      "/usr/bin)",
+      {.visibility = visibility::user},
+      "/usr/bin")
```

Comment:
Two things:
* I would have them provide the full path to `rpk` so `/usr/bin/rpk` rather than just the directory it contains
* There should be a check
  *  It exists
  * It's executable
  * RP can execute it

Reply:
I agree, some of those checks may be weird to implement in the config store though. I'll see what I can do.